packages/form-layout/theme/lumo/vaadin-form-item-styles.js

```diff
@@ -8,7 +8,7 @@ registerStyles(
   'vaadin-form-item',
   css`
     :host {
-      --vaadin-form-item-row-spacing: 0;
+      --vaadin-form-layout-row-spacing: 0;
```

Comment:
This change makes it impossible to configure the row gap on the form-layout element:

```css
vaadin-form-layout {
  --vaadin-form-layout-row-spacing: 2em;
}
```

I guess we should keep `--vaadin-form-item-row-spacing: 0` here and set `--vaadin-form-layout-row-spacing: 0;` on the form-layout element instead.

Reply:
Done.